src/Controller/Traits/RegisterTrait.php

```diff
@@ -98,9 +104,14 @@ public function register()
         }
 
         $userSaved = $usersTable->register($user, $requestData, $options);
-        if (!$userSaved) {
+        if (!$userSaved && Configure::read('Users.Registration.ShowVerboseError')) {
+            $errors = \Collection($user->getErrors())->unfold()->toArray();
```

Comment:
If no error messages show the default one. 

Config should be in camelCase form, showVerboseError instead of ShowVerboseError